packages/ldap-integration/webpack.common.js

```diff
@@ -0,0 +1,52 @@
+const nodeExternals = require('webpack-node-externals');
+const path = require('path');
+const fs = require('fs');
+const Dotenv = require('dotenv-webpack');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
```

Comment:
Why not call just call it `cleanWebpack`?

Reply:
https://github.com/johnagan/clean-webpack-plugin

Based on the documentation we are importing like this.